patches/chrome-browser-payments-BUILD.gn.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/chrome/browser/payments/BUILD.gn b/chrome/browser/payments/BUILD.gn
+index 9bf49781cc4159c1c936f152c5accdf806f0941d..9fe8cf158358bb8b240ff915c9aff97a458432ff 100644
+--- a/chrome/browser/payments/BUILD.gn
++++ b/chrome/browser/payments/BUILD.gn
+@@ -3,6 +3,7 @@
+ # found in the LICENSE file.
+ 
+ import("//build/config/chromeos/ui_mode.gni")
++import("//brave/test/exclusions.gni")
+ 
+ source_set("browser_tests") {
+   testonly = true
+@@ -77,6 +78,7 @@ source_set("browser_tests") {
+   if (is_chromeos_ash) {
+     deps += [ "//ash/public/cpp/external_arc:external_arc" ]
+   }
++  sources -= brave_chromium_browser_test_exclusions_payments
```

Comment:
same as other PR, can we just use chromium_src overrides to effectively remove these files?

Reply:
As mentioned in https://github.com/brave/brave-core/pull/7901#discussion_r587358076, that would imply emptying quite a bunch of files but, in the other hand, I think would avoid having to maintain all the `if`s I had to use in the [`browser_tests_exclusions.gni`](https://github.com/brave/brave-core/pull/7953/files#diff-ace633b17a0b78da6fdc6d4dc5a3d3100e2d092e79bf3b1bca310070ea8a634c) file, so worth a go.

I'll do this change once I finish doing the same for `unit_tests`, thanks!